modules/core/include/opencv2/core/utils/logger.hpp

```diff
@@ -16,6 +16,7 @@
 //
 //! @{
 
+namespace cv {
 namespace utils {
 namespace logging {
```

Comment:
Good catch!

Reply:
I'm surprised that one hasn't caused problems for other peoples builds.